### PR TITLE
RATIS-2188. Update website with Ratis 3.1.2 release

### PR DESCRIPTION
**Pull request by OneSizeFitsQuorum**
as title said

### CODE REVIEW

**Comment by OneSizeFitsQuorum** [comment]
@szetszwo PTAL

File: content/post/3.1.2.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: Release 3.1.2 is available
+date: 2024-11-11
```
Comment (adoroszlai) [inline content/post/3.1.2.md:3]:
```suggestion
date: 2024-11-13
```

**Review by adoroszlai — COMMENTED** [review]
Thanks @OneSizeFitsQuorum for the patch.

I think we should also remove 3.1.1 and 3.1.0 from the table.

```diff
--- content/post/3.1.0.md
+++ content/post/3.1.0.md
@@ -2,7 +2,7 @@
 title: Release 3.1.0 is available
 date: 2024-06-29
 type: release
-linked: true
+linked: false
 ---
 <!---
   Licensed under the Apache License, Version 2.0 (the "License");
--- content/post/3.1.1.md
+++ content/post/3.1.1.md
@@ -2,7 +2,7 @@
 title: Release 3.1.1 is available
 date: 2024-09-26
 type: release
-linked: true
+linked: false
 ---
 <!---
   Licensed under the Apache License, Version 2.0 (the "License");
```

File: content/post/3.1.2.md (hunk 1 above)
Comment (OneSizeFitsQuorum) [reply content/post/3.1.2.md]: Fixed

**Review by adoroszlai — APPROVED** [review]

**Comment by adoroszlai** [comment]
Thanks @OneSizeFitsQuorum for updating the patch.

**Comment by szetszwo** [comment]
@OneSizeFitsQuorum , thanks a lot for working on this!

@adoroszlai , thanks for reviewing and merging this!  Also, thanks for moving the release artifacts in svn!

**Comment by OneSizeFitsQuorum** [comment]
@adoroszlai @szetszwo Thanks a lot for reviewing this pr!

**Pull request merged**